core/src/test/scala/com/salesforce/op/stages/impl/tuning/DataSplitterTest.scala

```diff
@@ -43,6 +43,7 @@ class DataSplitterTest extends FlatSpec with TestSparkContext with SplitterSumma
 
   val seed = 1234L
   val dataCount = 1000
+  val MaxTrainingSampleDefault = 1E6.toLong
```

Comment:
I would give this a different name so we don't confuse it with the actual default in SplitterParams